README.md

```diff
@@ -7,7 +7,7 @@ Sunrise Java :sunrise:
 The next generation shop template.
 
 * Demo: https://sunrise.commercetools.com
-* [Documentation](https://github.com/commercetools/commercetools-sunrise-java/blob/master/ONBOARDING.md)
+* [Documentation](https://github.com/commercetools/commercetools-sunrise-java/blob/master/manual/)
```

Comment:
I would prefer not absolute links for this, so we can use the links in branches
```
[Documentation](manual/)
```

also works